hyperspy/tests/signal/test_tools.py

```diff
@@ -158,6 +158,12 @@ def test_print_summary(self):
         # Just test if it doesn't raise an exception
         self.signal._print_summary()
 
+    def test_print_summary_statistics(self):
+        # Just test if it doesn't raise an exception
+        self.signal.print_summary_statistics()
+        if self.signal._lazy:
+            self.signal.print_summary_statistics(rechunk=False)
```

Comment:
This method uses a lot of memory when I guess it shouldn't.

Reply:
That can happen when the chunking is not optimal (unsurprising since we don't rechunk) but, are you sure? The signal is rather small.

Reply:
I have tested it using the example in #1986/

Reply:
I've just run it and memory consumption doesn't seem aberrant to me. This method is very simply, and if there is an issue with it I guess that it must be unrelated to HyperSpy